db/migrate/20231129182201_add_author_id_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToPosts < ActiveRecord::Migration[7.1]
+  def change
+    add_column :posts, :author_id, :integer
+    add_index :posts, :author_id
+  end
+end
```

Comment:
- [ ] [optional] Kindly consider removing this migration file, you have already created the author_id column in the `20231129101210_create_posts` file